plugins/plants.cpp

```diff
@@ -148,12 +149,12 @@ command_result df_createplant (color_ostream &out, vector <string> & parameters)
     else
     {
         plant->hitpoints = 100000;
-        plant->flags.bits.is_shrub = 1;
+        plant->type = df::plant_type::DRY_PLANT;
     }
-    // for now, always set "watery" for WET-permitted plants, even if they're spawned away from water
+    // for now, always set  to WET_TREE for WET-permitted plants, even if they're spawned away from water
     // the proper method would be to actually look for nearby water features, but it's not clear exactly how that works
     if (plant_raw->flags.is_set(plant_raw_flags::WET))
-        plant->flags.bits.watery = 1;
+        plant->type = df::plant_type::WET_TREE;
```

Comment:
This needs to be either `WET_TREE` or `WET_PLANT`

Reply:
thanks for catching that! Fixed in 97153cf09